be/src/olap/rowset/segment_v2/column_reader.h

```diff
@@ -188,5 +191,39 @@ class FileColumnIterator : public ColumnIterator {
     rowid_t _current_rowid = 0;
 };
 
+// This iterator is used to read default value column
+class DefaultValueColumnIterator : public ColumnIterator {
+public:
+    DefaultValueColumnIterator(const TabletColumn& tablet_column)
+        : _tablet_column(tablet_column) { }
+
+    ~DefaultValueColumnIterator() { }
```

Comment:
either `delete[] _default_value` in destructor or change the type to string/faststring. I'd prefer the latter.

Reply:
ok